plugins/ros/src/components/riScInfo/riScStatus/RiScStatusComponent.tsx

```diff
@@ -70,12 +70,74 @@ const RiScPublishDialog = ({
   );
 };
 
+interface RiScMigrationDialogProps {
+  openDialog: boolean;
+  handleCancel: () => void;
+  handleUpdate: () => void;
+}
+
+const RiScMigrationDialog = ({
+  openDialog,
+  handleCancel,
+  handleUpdate,
+}: RiScMigrationDialogProps): ReactComponentElement<any> => {
+  const { t } = useTranslationRef(pluginRiScTranslationRef);
+
+  const [saveMigration, setSaveMigration] = useState<boolean>(false);
+
+  const handleCheckboxInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSaveMigration(event.target.checked);
+  };
+
+  return (
+    <Dialog open={openDialog}>
+      <DialogTitle>{t('migrationDialog.title')}</DialogTitle>
+      <DialogContent>
+        <Alert severity="info" icon={false}>
+          <FormControlLabel
+            control={
+              <Checkbox
+                color="primary"
+                checked={saveMigration}
+                onChange={handleCheckboxInput}
+              />
+            }
+            label={t('migrationDialog.checkboxLabel')}
+          />
+        </Alert>
+      </DialogContent>
+      <DialogActions sx={dialogActions}>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={handleUpdate}
+          disabled={!saveMigration}
+        >
+          {t('dictionary.confirm')}
+        </Button>
+        <Button variant="outlined" color="primary" onClick={handleCancel}>
+          {t('dictionary.cancel')}
+        </Button>
+      </DialogActions>
+    </Dialog>
+  );
+};
+
 const RosAcceptance = ({
   status,
+  migration,
 }: {
   status: RiScStatus;
+  migration?: boolean;
 }): React.JSX.Element => {
   const { t } = useTranslationRef(pluginRiScTranslationRef);
+  if (migration) {
+    return (
+      <Typography paragraph style={{ fontWeight: 700 }} variant="subtitle1">
```

Comment:
Smak og behag kanskje, men det er kanskje lettere å bruke vår egen subtitle1-definisjon fremfor den som er innebygd i temaet. Dersom en designer med tid ønsker å endre hvordan subtitle1 ser ut må vi enten endre theme i backstage (som kan bli slit med kartverket.dev) eller bare bruke våre egne stiloppskrifter på det, som jeg har lagt inn i typography.ts

```suggestion
      <Typography paragraph sx={subtitle1}>
```

Reply:
Ah, det gir mening. Jeg brukte samme styling som resten av innholdet i komponenten allerede hadde. Jeg har oppdatert de også nå